prowler/providers/aws/services/rds/rds_cluster_multi_az/rds_cluster_multi_az.py

```diff
@@ -0,0 +1,22 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_cluster_multi_az(Check):
+    def execute(self):
+        findings = []
+        for db_cluster in rds_client.db_clusters:
```

Comment:
To iterate over a dict in Python I recommend you to use `items()` it will retrieve the key and the value in the same iteration.
```suggestion
        for db_cluster_arn, db_cluster in rds_client.db_clusters.items():
```

Reply:
Thanks for your advice! 😄